### ISSUE TITLE
Replace with the BGRT spinner plymouth theme

### ISSUE THREAD

**Issue by ghost** [Vanilla-OS/plymouth-theme-vanilla#17]
**This is what it looks like in Fedora _(please ignore the old Fedora logo at the bottom as it has been replaced by the newer one)_:**
![plymouth-bgrt-spinner-fedora](https://github.com/user-attachments/assets/ed6c050c-577e-463b-8b87-428c9d5b4559)

As you can kinda see it shows the firmware/ BGRT logo in the middle, with an adwaita spinner in the middle and the distribution logo on the bottom. This, IMO looks much more professional and shows the firmware logo or BGRT. You'll have to just add the Vanilla OS distribution logo (which will be shown on the bottom part) in the theme files, package it in the repos, and replace the existing Plymouth theme package with this one.

**Comment by mirkobrombin** [comment]
I am personally against this, if OEM wants to show their brand, they co go this way but Vanilla OS has its own.

**Comment by ghost** [comment]
> I am personally against this, if OEM wants to show their brand, they co go this way but Vanilla OS has its own.

Is the package for this theme already installed on VanillaOS?

**Comment by axtloss** [comment]
Yes, its in `/usr/share/plymouth/themes/spinner`

**Comment by ghost** [comment]
> Yes, its in `/usr/share/plymouth/themes/spinner`

Thanks! How do I switch to it, tho?

**Comment by axtloss** [comment]
edit `/etc/plymouth/plymouthd.conf` and replace `Theme=vanilla-bgrt` with `Theme=spinner`, you may have to regenerate the initramfs with `abroot update-initramfs`

**Comment by ghost** [comment]
> edit `/etc/plymouth/plymouthd.conf` and replace `Theme=vanilla-bgrt` with `Theme=spinner`, you may have to regenerate the initramfs with `abroot update-initramfs`

Thanks again!

**Comment by ghost** [comment]
> edit `/etc/plymouth/plymouthd.conf` and replace `Theme=vanilla-bgrt` with `Theme=spinner`, you may have to regenerate the initramfs with `abroot update-initramfs`

There is no `plymouth` folder on my install.

**Comment by axtloss** [comment]
run `host-shell`, then try again

**Comment by ghost** [comment]
> run `host-shell`, then try again

I followed the instructions and rebooted but only the spinner showed up on the splash screen, no distribution logo or anything.

**Comment by axtloss** [comment]
yes, you'll have to add the distro logo yourself, im not sure how to do that
we disable the oem logo through a kernel argument, run `abroot kargs edit` and remove `bgrt_disable`

**Comment by taukakao** [comment]
I agree that our current plymouth theems is better, especially since it goes cleanly over to gdm without changing color.

We can add a spinner but I don't really see a benefit in that.
